src/components/common/Extensions/UserHub/partials/TransactionsTab/TransactionsTab.tsx

```diff
@@ -60,20 +58,15 @@ const TransactionsTab: FC<TransactionsProps> = () =>
         <p className="heading-5 mb-2.5 sm:mb-0.5">
           {formatText({ id: 'transactions' })}
         </p>
-        <div
-          id="transactionsListContainer"
```

Comment:
Are we sure this id is not being used anywhere else? Can we safely remove this? I see that there's a hook that uses it

Reply:
I saw that hook which uses it, TransactionsTab.css contains transactionsListContainer. but apart from that, I can't find anywhere else it's being used

Reply:
Removed the css import as that was bringing in the styles for the scrollbar, then added tailwind css for the container which meant the css file was no longer needed. issue has been resolved and files have been cleaned up.

Reply:
@warrencolliver it sounds like it is being used elsewhere then, are you sure it can be safely removed? If the answer is yes then we need to remove everything that previously relied on this id, including the hook you mentioned.

Reply:
> @warrencolliver it sounds like it is being used elsewhere then, are you sure it can be safely removed? If the answer is yes then we need to remove everything that previously relied on this id, including the hook you mentioned.

@jakubcolony The hook cannot be safely removed. Without this hook the transactions tab will not show more than 10 transactions at a time. this wasn't picked up previously as it wasn't tested with more than 10 transactions. because of this I've kept the id in there so the hook is still functional